tests/test_fastavro.py

```diff
@@ -1985,6 +1985,43 @@ def test_return_record_name_with_named_type_in_union():
     assert records == rt_records
 
 
+def test_return_record_with_named_type_in_union():
```

Comment:
Can we add another test where there is a union of `null`, `record1`, and `record2` and someone sets `return_record_name` to `True`? The code as is looks like it would consider that a nullable union and override the `return_record_name` which we wouldn't want.

Reply:
I believe you are right. I could add a len == 2 in the is_nullable test